blog/district-c-collaboration/index.md

```diff
@@ -0,0 +1,201 @@
+---
+title:
+  'Student insights - the Hitachi Energy and district C technical writing
+  initiative'
+authors: niewiarowski
+date: '2024-10-17'
+tags:
+  - 'bez-kategorii'
+  - 'dobre-praktyki'
+  - 'warsztat'
+  - 'praktyki'
+  - 'articles-in-english'
+coverImage: 'group_photo.jpg'
+---
+
+In 2023, the Hitachi Energy Technical Writing Team in Krakow participated in
+District C, a project run by the University of North Carolina Wilmington (UNCW)
+to inspire the next generation of energy professionals.
+
+<!--truncate-->
+
+District C lets students explore a real-life company\'s business operations.
+They learn about the unique aspects of an industry hands-on. In the end,
+students offer practical advice to the given problem during a pitch meeting,
+often their first experience presenting a business case to stakeholders. Read
+the story of the collaboration between District C and Hitachi Energy in Krakow,
+written from the perspective of Andrzej, a member of the Technical Writing Team.
+
+![](images/students.png) **Participants brainstorm over Zoom.**\_
+
+Our Technical Writing Team consisted of four people, out of which two were new
+joiners. Each of us had a different set of experiences and knowledge to share.
+It was in our best interest to think about how to collaborate better and align
+our efforts.
+
+The American and Polish students participating in the project could interview
+the Technical Writing Team about our team's role at Hitachi Energy. Students
+used design thinking principles to examine our work environment, aiming to
+analyze, find opportunities, gather suggestions, and present them.
+
+We shared our experiences that gave the students an idea of what makes technical
+writing rewarding and challenging:
+
+- Complex products and systems
+- Volume of information
+- Gathering insights from experts and seeking feedback
+- Building knowledge about the Hitachi Energy products
+
+![](images/soap_box.jpg) **Lance, Quinn, and Andrzej tell a story about the
+project at the soap! content conference in June, 2023.**
+
+## How it was done
+
+First, there were two rounds of Zoom meetings and presenting information.
+
+The members of the Hitachi Energy Team explained what it is like to be a
+Technical Writer and why this role requires specialized expertise. Concise and
+clear data gave insight into the organization's daily work and critical focus
+areas.
+
+Then, students pondered what was said and came back with additional questions to
+dig deeper. Each student group had a specific amount of time to ask strategic
+questions. These were intense, focused sessions in which both sides tried to use
+the available time as best they could.
+
+At that point, this exercise was already beneficial to the Hitachi Energy Team.
+We all knew the specifics of our daily operations well. Still, it was a useful
```

Comment:
Having said that, it was a useful exercise to describe our role to someone...

Reply:
Na razie usunalem "At that point" na poczatku.

"After everything" mi tu nie pasuje. Te wywiady ze studentami traktuje jako "mini" cwiczenie w ktorym choc znamy sie na tym co robimy, to opowiedzenie tego komus z zewnatrz nam pomaga. Na razie tak zostawie, mozemy jeszcze o tym pogadac.